besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1538,6 +1578,11 @@ private PrivacyParameters privacyParameters() {
       privacyParametersBuilder.setPrivateKeyPath(privacyMarkerTransactionSigningKeyPath);
       privacyParametersBuilder.setStorageProvider(
           privacyKeyStorageProvider(keyValueStorageName + "-privacy"));
+      if (isOrionTlsEnabled) {
+        // TODO should this use TlsConfiguration
+        privacyParametersBuilder.setTlsConfiguration(
```

Comment:
unit test for the tls config

Reply:
after some consideration, leaving this for AT since the privacyParameters.build actually constructs the enclave